portfolio/src/main/webapp/script.js

```diff
@@ -122,5 +122,28 @@ function deleteAllComments(){
     console.log('All comments are deleted');
 
 }
+getGoogleAPILink();
+
+function createGoogleMap() {
+    console.log("going");
+  var map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: -33.864933730676675, lng: 151.1961065202642}, zoom: 8});
+
+      map.setMapTypeId('satellite');
+      
+    const googleSydneyOffice = new google.maps.Marker({
+    position: {lat: -33.864933730676675, lng: 151.1961065202642},
+    map: map,
+    title: 'Google Sydney office'
+  });
+}
```

Comment:
Indentation in this function is inconsistent

Reply:
(this is something that would be caught by a formatter in future, not a big deal)